spec/index.html

```diff
@@ -406,6 +406,7 @@ <h2>Simple Interpretations</h2>
               set of sets of pairs &lt; x, y &gt; with x and y in IR .</p>
         <p>4. A mapping IS from IRIs into (IR union IP)</p>
         <p>5. A partial mapping IL from literals into IR </p>
+        <p>6. A partial injective mapping RE from IR x IP x IR into IR, called the interpretation of triple terms. </p>
```

Comment:
The 'partial' needs to be changed to 'total' to allow entailments containing triple terms that do not appear in the graph in the consequent and to match what is done for IRIs.